Modules/Filtering/DisplacementField/test/itkTimeVaryingVelocityFieldIntegrationImageFilterTest.cxx

```diff
@@ -64,7 +64,7 @@ itkTimeVaryingVelocityFieldIntegrationImageFilterTest(int, char *[])
   index.Fill(0);
   VectorType displacement;
 
-  auto = IntegratorType::New();
+  auto inverseIntegrator = IntegratorType::New();
```

Comment:
This commit should be squashed with the one which introduced the use of `auto` here.